fordarnold/Papel
# #171254001 Start final code cleanup

#### What does this PR do?
Do final code cleanup for the app UI (Challenge 1 ADC).

#### Description of Task(s) to be completed?
- Add demo URL in project README file.
- Add smooth scrolling on public pages.
- Refine design of the sign in form.
- Fix design of signed-in user's top navigation bar.
- Move small-sized forms to a modal dialog UI.
- Fix broken form input.
- Delete redundant/unnecessary files (general cleanup).
- Update design of card widgets.

#### How should this be manually tested?
Browse different pages of the app, starting from the landing page, to see the latest updates.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#171254001

#### Screenshots (if appropriate)

![screenshot-sign-in](https://user-images.githubusercontent.com/1271441/74779903-1bf0bf80-52a7-11ea-8cd1-7004c412bb3a.png)

![screenshot-account-dashboard](https://user-images.githubusercontent.com/1271441/74780002-5c503d80-52a7-11ea-8a08-8f7ca71b8ae2.png)

#### Questions:
